packages/flame_studio/lib/src/widgets/toolbar/toolbar.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flame_studio/src/core/settings.dart';
+import 'package:flame_studio/src/widgets/flame_studio.dart';
+import 'package:flutter/widgets.dart';
+
+class Toolbar extends StatelessWidget {
+  const Toolbar({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final settings = Settings.of(context);
+    final height = settings.toolbarHeight;
+    final gap = height * 0.10;
+
+    Widget _wrapButton(Widget button) {
```

Comment:
Ideally this would be an internal, stateless widget instead.

Reply:
done